spark/src/test/java/org/apache/iceberg/spark/SparkTestBase.java

```diff
@@ -73,9 +73,6 @@ public static void startMetastoreAndSpark() {
 
   @AfterClass
   public static void stopMetastoreAndSpark() {
-    if (catalog != null) {
-      catalog.close();
-    }
```

Comment:
Should we make this consistent with other tests and set the catalog to null?

Reply:
It is nulled out in the next line.